railties/lib/rails/generators/app_base.rb

```diff
@@ -286,7 +286,7 @@ def database_gemfile_entry # :doc:
 
         gem_name, gem_version = database.gem
         GemfileEntry.version gem_name, gem_version,
-          "Use #{options[:database]} as the database for Active Record"
+          "Use #{database.adapter} as the database for Active Record"
```

Comment:
Why was this needed?

Reply:
> Why was this needed?

I believe this line adds a comment in the generated Gemfile, so, I kept it consistent: regardless if you use `postgres` or `postgresql`, it always outputs: Use postgresql as the database for Active Record.